servicetalk-http-api/src/main/java/io/servicetalk/http/api/DefaultHttpSetCookie.java

```diff
@@ -447,6 +479,35 @@ public CharSequence subSequence(final int start, final int end) {
         }
     }
 
+    @Nullable
+    private static SameSite fromSequence(CharSequence cs, int begin, int end) {
+        final int length = end - begin;
+        if (length == Lax.toString().length()) {
```

Comment:
nit: you could optimize this with a switch statement:

```
switch (length) {
    // lax
    case 3: 
       ....
    // none
    case 4:
       ....
    // strict
    case 6:
       ...
    default:
        return null;

}

```

Reply:
done